python-lib/dku_timeseries/timeseries_helpers.py

```diff
@@ -156,7 +156,10 @@ def convert_to_rolling_compatible_time_unit(time_step, time_unit):
         return 7 * time_step, 'days'
     elif time_unit == 'months' or time_unit.startswith("M"):
         return 30 * time_step, 'days'
-    elif time_unit == 'years' or time_unit.startswith("A"):
+    elif time_unit == 'quarters' or time_unit.startswith("Q"):
+        return 91 * time_step, 'days'
+    # There is no half year frequency, those are inferred as two quarters instead in infer_frequency()
+    elif time_unit == 'years' or time_unit.startswith("Y") or time_unit.startswith("A"):
```

Comment:
suggestion : we could take this opportunity to also support business days 
![Screenshot 2025-01-16 at 18 32 08](https://github.com/user-attachments/assets/0455f812-1614-456a-8696-f01d2d990b8a)
I guess we can assume 1 business day = 7/5 days?

Reply:
How did you manage to business days as your infered frequency ? :-D putting monday to friday repeatedly in the dataset ?

Reply:
I agree with the proposition otherwise, i'm just surprised it occured 😂

Reply:
I used the resampling recipe to have a frequency of 1 business day => https://staging-design.qa.managedinstances.dkucloud-dev.com/projects/MARINETIMESEROES/recipes/compute_business_days/

Reply:
I'm having a go at this change, and in fact I don't think we should proceed with 7/5 days - the way the plugin works, if you window for example over 4 days with a business days dataset, we will in the end window over `ceil(4 * 5/7) `= 3 days. 

Considering the user may not even realize we inferred business days in the first place, this seems very unintuitive, and I think I'd rather raise a clean error, WDYT ?

Reply:
yes, I agree!